### ISSUE TITLE
Add Versatile Video Coding (VVC) … somewhen

### ISSUE THREAD

**Issue by Schweinepriester** [Fyrd/caniuse#5065]
- <https://en.wikipedia.org/wiki/Versatile_Video_Coding>
- <https://www.bbc.co.uk/rd/blog/2019-05-av1-codec-streaming-processing-hevc-vvc>
- <https://jvet.hhi.fraunhofer.de/>
- (german) <https://www.heise.de/newsticker/meldung/HEVC-Nachfolger-VVC-H-266-Gleiche-Videoqualitaet-bei-halber-Groesse-4512946.html>

Wiki:
> End of 2020: First official standard

A bit early, thus "somewhen" ;)

**Comment by Schweinepriester** [comment]
- <https://9to5mac.com/2020/07/06/h-266-vvc-codec-released-successor-h-265-hevc-higher-quality-video-capture-ios-iphone/>
- <https://newsletter.fraunhofer.de/-viewonline2/17386/465/11/14SHcBTt/V44RELLZBp/1>
- [Wiki again](https://en.wikipedia.org/wiki/Versatile_Video_Coding):

   > **Versatile Video Coding** (**VVC**), also known as **H.266**, **MPEG-I Part 3** and **Future Video Coding** (**FVC**), is a [video compression standard](https://en.wikipedia.org/wiki/Video_coding_format "Video coding format") finalized July 6th, 2020 by the Joint Video Experts Team (JVET)

**Comment by haudiobe** [comment]
Can this be added. CTA WAVE adopted this in their content spec.

**Comment by mqudsi** [comment]
I think H.266/VVC and its sibling EVC should both be added to the chart, even now with all versions of all browsers all represented by red blocks with no support. 

It's natural for developers to look up a codec and see if it's supported and we shouldn't punish precocious devs by turning up no results when we know exactly what they mean and what they want to know when they search h.266, vvc, or evc.

**Comment by Slapbox** [comment]
Any update on this? While it may be early for mainstream uses, niche uses that require extreme compression rates need to know what their options are.

**Comment by tats-u** [comment]
First of all, the discussion will not begin until GPUs support hardware decoding.
Software decoding won't be implemented due to patents.

**Comment by Slapbox** [comment]
Thanks for the explanation @tats-u. Wouldn't it be better though, to add it and say you can't use it yet and add that as a note?

**Comment by tats-u** [comment]
@Slapbox We can do them only after at least one of Intel, Nvidia, AMD, and Qualcomm start supporting the VVC hardware decoding, or Apple announces a plan to support VVC on their OSes or Safari.
I think it is too early to do them now.

**Comment by tats-u** [comment]
@Slapbox We can do them only after at least one of Intel, Nvidia, AMD, and Qualcomm start supporting the VVC hardware decoding, Apple announces a plan to support VVC on their OSes or Safari, or VVC Video Extension starts sold in Microsoft Store like HEVC one.
I think it is too early to do them now.

**Comment by mqudsi** [comment]
@tats-u I've already mentioned my opinion above, but never got an actual response. 

Start with the question of who is the audience of caniuse and what is the goal of the site. It is _not_ MDN. It is a developer resource. If there's a video codec that has gained enough popularity for developers to wonder about, it is natural for it to be listed on the site with all browsers X and a note that says "no browser has announced any plans to support vvc at this time" or whatever. There's really no downside that I can see, since it only comes up when you search for the particular keywords. 

Right now, the experience is extremely suboptimal: a developer arrives at CIU and searches "vvc" and gets no results. **They don't know if this is because VVC isn't supported anywhere or if it's because CIU simply hasn't listed VVC support anywhere.** 

Explicitly listing it with zero browser support makes it clear to the developer which of those two possibilities is the matter.

**Comment by tats-u** [comment]
@mqudsi you are right. We have to convince visitors that it is too hard for browsers to support VVC by themselves for the above reasons.
